src/web/Html.tsx

```diff
@@ -341,7 +341,7 @@ export const Html: ForwardRefComponent<HtmlProps, HTMLDivElement> = /* @__PURE__
             }
             el.style.width = size.width + 'px'
             el.style.height = size.height + 'px'
-            el.style.perspective = isOrthographicCamera ? '' : `${fov}px`
+            el.style.transform = isOrthographicCamera ? '' : `perspective(${fov}px)`
```

Comment:
This will need to be the first transform on "transformOuterRef" below, rather than a transform on "el".